manuscript/manuscript.Rmd

```diff
@@ -19,12 +19,27 @@ authors:
     location: Berkeley, California
     email: cboettig@berkeley.edu
 abstract: |
-  Enter the text of your abstract here.
+  Can machine learning really help us make better decisions about a changing planet?
+  In this paper, we illustrate and discuss the potential of a promising 
+  corner of machine learning known as _reinforcement learning_ (RL) to help tackle the
+  most challenging conservation decision problems. RL is uniquely
+  well suited to conservation and global change challenges for three reasons: (1)
+  rather than passively modeling observed data, RL is explicitly designed around 
```

Comment:
I think you could take out "rather than passively modeling observed data", not lose anything and be crisper. I think this point is also implied in the third point.